docs/DeveloperGuide.md

```diff
@@ -155,7 +155,7 @@ _to be updated_
 
 ### Search-related features
 
-Format: `search <userAction> <keywords>`
+Format: `search <commandAction> <keywords>`
```

Comment:
Thanks for making it consistent with the user guide! Good Job!

Reply:
Thank you!